<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome::WelcomeMode</class>
 <widget class="QWidget" name="Welcome::WelcomeMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>690</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#Welcome--WelcomePage {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}

QToolButton, QPushButton, QComboBox {
    border-image: url(:/welcome/images/btn_26.png) 4;
    border-width: 4;
    padding: 0px 6px;
    font-size: 12px;
}

*{
    color: black;
}

 QComboBox QAbstractItemView {
    background-color:white;
}

 QComboBox::down-arrow {
     image: url(:/welcome/images/combobox_arrow.png);
 }

QComboBox:drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     border-left-style: none; 
     border-top-right-radius: 1px;
     border-bottom-right-radius: 1px;
}

QToolButton:hover, QPushButton:hover, QComboBox:hover {
    border-image: url(:/welcome/images/btn_26_hover.png) 4;
}

QToolButton:disabled, QPushButton:disabled, QComboBox::disabled {
	color:gray;
}

QToolButton:pressed, QPushButton:pressed{
    border-image: url(:/welcome/images/btn_26_pressed.png) 4;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="gradientWidget" native="true">
     <property name="styleSheet">
      <string>#gradientWidget {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QFrame" name="mainFrame">
        <property name="minimumSize">
         <size>
          <width>662</width>
          <height>515</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>662</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#mainFrame {
    border-image: url(:/welcome/images/background_center_frame.png) 4;
    border-width: 4;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="headerFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>649</width>
             <height>74</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>649</width>
             <height>89</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>#headerFrame {
    border-image: url(:/welcome/images/center_frame_header.png) 0;
    border-width: 0;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="navFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #838383,
	stop: 0.4 #707070,
	stop: 0.401 #636363,
	stop: 1 #4a4a4a);
	border: 0px solid black;
	border-top: 1px solid #303030;
	border-bottom: 1px solid #202020;
	color: white;
	height:20px;
}

QToolButton:hover {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #909090,
	stop: 0.4 #808080,
	stop: 0.401 #707070,
	stop: 1 #555555);
}

QToolButton:checked, QToolButton:checked:pressed {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #ffffff,
	stop: 0.4 #eeeeee,
	stop: 0.401 #e2e2e2,
	stop: 1 #dddddd);
	color: black;
	border-top: 1px solid #606060;
	border-bottom: 1px solid #404040;
}

QToolButton:pressed {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #383838,
	stop: 0.2 #404040,
	stop: 0.201 #484848,
	stop: 1 #505050);
}


</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>17</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="helpUsLabel">
             <property name="text">
              <string>Help us make Qt Creator even better</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>6</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="feedbackButton">
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">height:19px;</string>
             </property>
             <property name="text">
              <string>Feedback</string>
             </property>
             <property name="icon">
              <iconset resource="welcome.qrc">
               <normaloff>:/welcome/images/feedback_arrow.png</normaloff>:/welcome/images/feedback_arrow.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="bottomVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="topVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="welcome.qrc"/>
 </resources>
 <connections/>
</ui>
